sci_strategic_plugin/config/parameters.yaml

```diff
@@ -5,7 +5,7 @@ vehicle_decel_limit_multiplier : 0.65
 vehicle_accel_limit_multiplier : 0.65
 
 # Double: A buffer before of the stopping location which will still be considered a valid stop. Units in meters
-stop_line_buffer : 3.5
+stop_line_buffer : 10
```

Comment:
Is this acceptable? How does it compare with UC2?

Reply:
UC2 has stopping location buffer of 8 meters, which I think is comparable to this value. UC2 has a little more sophisticated guard for stopping case that allows it to stop closer (once it is decided to stop, uc2 plugin will keep it slowing, while uc1 may try to lane follow again in stopping distance). So another way to make it stop more consistently without changing the code was increasing this value.